doc/source/api.md

```diff
@@ -236,7 +236,7 @@ To build and pass your coregistration pipeline to {func}`~xdem.DEM.coregister_3d
 .. autosummary::
     :toctree: gen_modules/
 
-    coreg.workflows.dem_coregistration
+    DEM.coregister_3d
```

Comment:
This entire subsection "Quick coregistration" can be removed, as `DEM.coregister_3d()` is already listed further above in the API.